game/scripts/npc/abilities/chaos_knight_phantasm.txt

```diff
@@ -63,12 +63,12 @@
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "incoming_damage"                                 "200" //OAA
+        "incoming_damage"                                 "150" //OAA
```

Comment:
CK ultimate needs a cooldown reuduction aswell. 120 seconds is unusable for a carry. His ultimate needs more uptime. 90 seconds is more ideal. (I doubt your purchasing octarine core on chaos knight.)

Reply:
If you were balancing this game, every ultimate would have 30 seconds. This isnt battlerite. Stop.